.pre-commit-config.yaml

```
@@ -122,7 +122,7 @@ repos:
 
   # Check for spelling
   - repo: https://github.com/crate-ci/typos
-    rev: v1.34.0
+    rev: v1
```

```suggestion
    rev: v1.34.0
```
This is truly getting annoying. I hope the authors of the library change their tagging policy at some point so that we don't have to constantly revert these changes.